src/ar_node_worker.erl

```diff
@@ -445,6 +446,18 @@ process_new_block(#{ height := Height, cumulative_diff := CDiff } = StateIn, New
 			none
 	end.
 
+log_missing_txs(TXIDs, ReconstructedTXs) ->
+	case length(TXIDs) of
+		L when L > length(ReconstructedTXs) ->
+			MissingTXIDs = TXIDs -- lists:map(fun(TX) -> TX#tx.id end, ReconstructedTXs),
+			ar:info(
+				"Could not find these transactions from the new block: ~p",
```

Comment:
I think we should add the indep hash here as well. E.g.:

`ar:info([{transactions_missing_in_mempool_for_block, BH}, {missing_txs, TXs}]).`